packages/form-js-viewer/src/features/expression-language/FeelersTemplating.js

```diff
@@ -29,6 +65,37 @@ export default class FeelersTemplating {
 
     return evaluateFeelers(template, context, { debug, strict, buildDebugString });
   }
+
+
+  _extractExpressionsWithDepth(template) {
```

Comment:
What could help here is an example return via JSDocs to make it easier to understand what the expressions array looks like. Maybe even a `@type` declarator. 

Now that I read through the implementation it becomes clear, but that was my first impression on the start 😅

Reply:
Sure, will ask the AI to do this one :D